polyfill/lib/absolute.mjs

```diff
@@ -1,126 +1,150 @@
 import { ES } from './ecmascript.mjs';
 import { EPOCHNANOSECONDS, CreateSlots, GetSlot, SetSlot } from './slots.mjs';
```

Comment:
this change means `GetSlot` can use private fields instead, no?

Reply:
No, because there is no backward compatible way to do this.

Current plan:
 if WeakMap use it
 if Symbol use it
 fallback '_SLOTS' property

Reply:
for this in-proposal-repo polyfill tho, isn't back compat not an issue?